packages/lesswrong/lib/collections/users/custom_fields.ts

```diff
@@ -1002,7 +1002,7 @@ addFieldsDict(Users, {
       nullable: true,
     }),
     optional: true,
-    canRead: ['sunshineRegiment', 'admins'],
+    canRead: ['sunshineRegiment', 'admins', 'guests'],
```

Comment:
Without this permission, reviewedByUserId is undefined unless you're logged in as an admin.